packages/next/src/build/swc/index.ts

```diff
@@ -669,8 +670,34 @@ function bindingToApi(
   class ProjectImpl implements Project {
     private readonly _nativeProject: { __napiType: 'Project' }
 
+    #poolCreated: Record<string, Array<Worker>> = {}
+
+    #poolScheduler?: ReturnType<typeof setInterval>
+
     constructor(nativeProject: { __napiType: 'Project' }) {
       this._nativeProject = nativeProject
+
+      if (binding.recvPoolCreation) {
+        this.#poolScheduler = setInterval(() => {
+          let poolOptions = binding.recvPoolCreation()
```

Comment:
That busy polling don't look good. Could we make recvPoolCreation async and await the pool creation instead.

Reply:
And when evaluate assets changed, should we create a new pool ?
https://github.com/utooland/next.js/blob/refactor/turbopack-node-napi/turbopack/crates/turbopack-node/src/evaluate.rs#L232.  If not, we can use just two promises to awaiting `postcss` and `webpack-loader` pool creation message.